tests/jenkins/run_test_installation_docs.sh

```diff
@@ -299,28 +317,53 @@ LINUX_PYTHON_GPU_END_LINENO=$(grep -n "END - Linux Python GPU Installation Instr
 set_instruction_set ${LINUX_PYTHON_GPU_START_LINENO} ${LINUX_PYTHON_GPU_END_LINENO}
 
 
-# mxnet/base-cuda9 is a simple Docker Image with 'nvidia/cuda:9.0-cudnn7-devel' and 'apt-get install sudo'.
+ubuntu_python_gpu_virtualenv()
+{
+    #$WORDTOREMOVE
+    echo
+    echo "### Testing Virtualenv ###"
+    echo "${virtualenv_commands}"
+    echo
+    #virtualenv_commands=${virtualenv_commands//$WORDTOREMOVE/}
```

Comment:
Remove

Reply:
Can you elaborate please?

Reply:
It's commented out, so we probably don't need it anymore, right?